tests/ignite/distributed/test_auto.py

```diff
@@ -42,6 +50,8 @@ def _test_auto_dataloader(ws, nproc, sampler_name=None, dl_type=DataLoader):
     if isinstance(dataloader, DataLoader):
         assert dataloader.pin_memory == ("cuda" in idist.device().type)
 
+    # test
```

Comment:
Can we remove this ?

Reply:
Done. I don't even know how it got there